### ISSUE TITLE
Widgets screen: Add the Cmd/Ctrl + S keyboard shortcut to save changes

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#25817]
**Describe the bug**

In the new Widgets screen, I expected to be able to use Cmd/Ctrl + S to save the edits to the widgets. Instead, pressing Cmd/Ctrl + S opens the browser "Save As..." dialog.

Given the lack of the keyboard shortcut, the only way to save changes for keyboard users is to navigate back by pressing Shift + Tab multiple times, potentially dozens of times depending on the amount of widgets and the state of the UI, to get to the Update button and activate it. This wouldn't be the best experience for keyboard users.

To make things faster and for consistency with the block editor, pressing Cmd/Ctrl + S should trigger saving. I'd tend to think this would be welcomed also by users who aren't exclusively keyboard users.

The lack of consistency with the behavior on the block editor could also be considered a "bug", as users who know the keyboard shortcut would likely expect it to work also on the Widgets screen.

**To reproduce**
1. Go to the Widgets screen
2. Make some edit
3. Press Cmd + S (on macOS) or Ctrl + S (on Windows)
4. See that the browser "Save As..." dialog opens

**Expected behavior**
The changes to be saved instead of triggering the "Save As..." dialog.

**Editor version (please complete the following information):**
- WordPress version: 5.6 trunk
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? plugin
- If the Gutenberg plugin is installed, which version is it? latest master 9.1.0
